toxbat/requirements.py

```diff
@@ -74,6 +58,25 @@ def is_enabled_env(env):
             env.recreate = True
     return config
 
+def all_nested_req_files(requirement_files):
+    """
+    Get all nested req file names for a list of requirements files.
+
+    :param iter requirements_files: list of requirements files to check.
+    :rtype: iter
+    """
+    for reqfile in requirement_files:
+        yield reqfile
+
+        if os.path.isfile(reqfile):
+            parent_dir = os.path.dirname(reqfile)
+            with open(reqfile) as f:
+                for line in f:
+                    if line.startswith("-r") or line.startswith("-c"):
```

Comment:
Could you please add a test for this functionality? If you don't have time I can do it too, let me know.